packages/legend-studio/src/components/editor/edit-panel/mapping-editor/MappingTestEditor.tsx

```diff
@@ -404,13 +412,13 @@ export const MappingTestInputDataBuilder = observer(
       (setImplementation: SetImplementation | undefined): void => {
         testState.setInputDataStateBasedOnSource(
           setImplementation
-            ? getMappingElementSource(setImplementation)
+            ? getMappingElementSource(setImplementation, plugins)
             : undefined,
           true,
         );
         hideClassMappingSelectorModal();
       },
-      [testState],
+      [testState, plugins],
```

Comment:
same as before. Here you can do `editorStore = useEditorStore()` instead of passing `plugins` in `MappingTestInputDataBuilder`